### DISCUSSION TITLE
all maps are broken

### DISCUSSION THREAD

**Discussion by yasashiiharuka** [ppy/osu#22541] [Q&A]
after reinstalling Windows, and moving the root folder back to where it was and then replacing the files, all the cards turned out to be broken.
this was not the case before.is there any way to restore them? the fact is that I have only a few maps. and it will not be difficult to restore these maps. but what if I have hundreds of thousands of maps as it was before this moment? it's just that my hard drive broke, and I had to buy a new one and reinstall the osu!  but although I could try to transfer all the cards from the old hard drive, but due to the fact that all the files are small, transferring almost 300 gigabytes would take a very, very long time.
So how can I fix this and what can I do to make sure it doesn't happen again? what if i redownload a lot of maps and they all break again?
[https://i.imgur.com/NsH4gqs.jpeg](url)

**Comment by yasashiiharuka** [comment]
oh, I think I get it. somehow, the game remembered all the maps that were before I transferred the game to another hard drive. and it just sees emptiness.

but nevertheless, I think this would not have happened if the game did not create the root folder in the C drive by default.

it would be better if Osu!Lazer also had the ability to choose the installation path of the game like the regular version of this game
[https://i.imgur.com/Wt2spvZ.png](url)

**Reply by mk56-spn** [reply]
This will probably come when Lazer gets a proper installer I'm guessing
